templates/shared/search/code/results.tmpl

```diff
@@ -24,7 +24,7 @@
 				{{else}}
 					<span class="file tw-flex-1">{{.Filename}}</span>
 				{{end}}
-				<a role="button" class="ui basic tiny button" rel="nofollow" href="{{$repo.Link}}/src/commit/{{$result.CommitID | PathEscape}}/{{.Filename | PathEscapeSegments}}">{{ctx.Locale.Tr "repo.diff.view_file"}}</a>
+				<a role="button" class="ui basic tiny button tw-mr-0" rel="nofollow" href="{{$repo.Link}}/src/commit/{{$result.CommitID | PathEscape}}/{{.Filename | PathEscapeSegments}}">{{ctx.Locale.Tr "repo.diff.view_file"}}</a>
```

Comment:
Use `<h4 class="flex-text-block ...">` (it seems better with proper gap)

Then the buttons in `flex-text-block` don't need `tw-mr-0` anymore.